webpack.config.js

```diff
@@ -42,7 +42,8 @@ module.exports = {
     },
     plugins: [
         new webpack.DefinePlugin({
-            SERVICE_URL: JSON.stringify(process.env.CW_URL)
+            SERVICE_URL: JSON.stringify(process.env.CW_URL),
+            ANALYTICS_TRACKING_ID: JSON.stringify(process.env.CW_GA_TRACKING_ID)
```

Comment:
this is being passed in to the webpack so that it can be referenced here:

it is not needed to get the GA working. it is in the webpack-rendered js so that GA can be programmatically turned off. You can't turn it off in JS without knowing the tracking ID.

Reply:
Ah I see it now, missed that